### ISSUE TITLE
When closing an issue, enforce a status

### ISSUE THREAD

**Issue by karlcow** [webcompat/webcompat.com#1305]
When the issue needs to be closed, the user click on  "close issue"

![capture d ecran 2017-01-26 a 16 08 50](https://cloud.githubusercontent.com/assets/505230/22322797/cbb88b6a-e3e1-11e6-9898-261dbe3c0458.png)

This button should create a menu where you choose the reason for closing. Then only activate, the closing. Maybe it ties to our discussion about the label editor #1240

**Comment by karlcow** [comment]
The why: This would minimize the number of issues being closed by webcompatbot with the wrong status.

**Comment by adamopenweb** [comment]
Good idea @karlcow. The list of reasons available should only be closed statuses: invalid, needsdiagnosis, incomplete, fixed, wontfix, worksforme.

**Comment by miketaylr** [comment]
Yeah, I really like Bugzilla's UI for this.

**Comment by zoepage** [comment]
@karlcow 
So, we could also add templates (we could also edit) to the reasoning or would it just add a label?

I love this idea btw.

**Comment by karlcow** [comment]
To make it simpler. Probably let's start with the close button triggering a reason. @miketaylr suggestion of bugzilla UI is good.

Close, choose:
* Wontfix
* duplicate -> request the number (webc / bugzilla)
* fixed
* invalid 
* incomplete

Then in a second stage we could add boilerplate choice for invalid, incomplete. Even for duplicate, I'm thinking, It's an HLS issue, a TP issue, a MarkFeel issue etc.
